en/app.ftl

```diff
@@ -0,0 +1,270 @@
+    This can happen if { -brand-name-relay } couldn’t connect to your email provider, or if your mailbox was full. We will try again on { $date }.
+banner-download-firefox-headline = { -brand-name-relay } is even better in { -brand-name-firefox }
+banner-download-firefox-copy = The { -brand-name-relay } extension for { -brand-name-firefox-browser } makes creating aliases even easier.
+banner-download-firefox-cta = Get { -brand-name-firefox }
+banner-download-install-extension-headline = Get the { -brand-name-relay } extension for { -brand-name-firefox }
+banner-download-install-extension-copy = The { -brand-name-relay } extension for { -brand-name-firefox-browser } makes using email aliases even easier.
+banner-download-install-extension-cta = Add { -brand-name-relay } to { -brand-name-firefox }
+banner-upgrade-headline = Upgrade to { -brand-name-relay-premium }
+banner-upgrade-copy = { -brand-name-firefox } { -brand-name-relay-premium } makes creating email aliases even easier, with custom alias domains and unlimited aliases.
+banner-upgrade-cta = Upgrade to { -brand-name-relay-premium }
+banner-choose-subdomain-headline = Choose your own domain
+banner-choose-subdomain-headline-aliases = Get your own custom alias domain
+banner-choose-subdomain-copy = You can choose a custom domain for your email aliases.
+banner-choose-subdomain-warning = Note: You cannot change your domain later
+banner-choose-subdomain-input-placeholder = Search domain
+banner-choose-subdomain-submit = Get Domain
+banner-pack-upgrade-headline-html = Upgrade to <strong>{ -brand-name-firefox } { -brand-name-relay-premium }</strong> to get more aliases
+banner-pack-upgrade-copy = With unlimited email aliases and your own email domain, { -brand-name-firefox } { -brand-name-relay-premium } helps you stay protected online.
+banner-pack-upgrade-cta = Upgrade Now
+
+# This string is followed by name (string) that the user chooses
+banner-choose-subdomain-label = Your domain is:
+
+# Variables:
+# $subdomain (url) - User-set subdomain
+banner-choose-subdomain-label = You can make up any address @{ $subdomain }
+
+## Error Messages
+
+# Variables:
+#   $number (string) - Maximum number of aliases a user can make on a free account. 
+error-premium-set-make-aliases = You must be a premium subscriber to make more than { $number } aliases
+error-premium-cannot-change-subdomain = You cannot change your subdomain
+error-premium-set-subdomain = You must be a premium subscriber to set a subdomain
+error-premium-set-create-subdomain = You must be a premium subscriber to create subdomain aliases
+error-subdomain-not-created = Subdomain could not be created, try something else
+error-subdomain-email-not-created = Email address with subdomain could not be created, try something else
+error-subdomain-select = You must select a subdomain before creating a subdomain aliases
+
+## Onboarding 
+
+onboarding-headline = Create your first alias, you have three ways…
```

Comment:
nitpick: add a space before the ellipsis.

Reply:
Fixed!